.circleci/wait-for-job-finish.sh

```diff
@@ -8,8 +8,8 @@ set -eo pipefail
 
 counter=0
 
-# Wait up to 25 minutes for job to finish. A job can run on multiple nodes: parallelism > 1
-while [ "$counter" -le 1500 ]; do
+# Wait up to 30 minutes for job to finish. A job can run on multiple nodes: parallelism > 1
```

Comment:
Time out should be >= than the longer running test which is `run-analysis-azure.ts` at this time. Test is set with max time out 30 min.